### PR TITLE
modified intro pages

### PR DESCRIPTION
**Pull request by AndrewTorrance**
### What is the context of this PR?
Trading as in title was being misused and holds information other than trading as. This change acknowledges that fact by changing the introduction from 'You are completing this for <Company> (Trading as <trading as name>)" to 'You are completing this for <Company> (<trading as name>)". i.e removing the 'trading as' string. It still only shows if the trading as parameter is set.

This is for all surveys that use the introduction not just MBS.

Additionally it changes the use of information in the header of MBS surveys by adding "unless otherwise stated." For example from "Data should relate to all sites in England, Scotland and Wales." to "Data should relate to all sites in England, Scotland and Wales unless otherwise stated."

### How to review 
Launch the surveys from go launcher . Note The change in trading as affects ALL business surveys .

### CODE REVIEW

**Review by nealedj — APPROVED** [review]

**Comment by nealedj** [comment]
Do not merge.

Waiting for stakeholder sign off on the other surveys re. the removal of "trading as" before merging.

**Comment by shahi645** [comment]
LGTM, wont approve as we are waiting on stakeholders

**Comment by shahi645** [comment]
Has been approved so ready to merge

**Review by shahi645 — APPROVED** [review]

**Pull request merged**
